pkg/providers/launchtemplate/suite_test.go

```diff
@@ -1538,14 +1560,32 @@ var _ = Describe("LaunchTemplates", func() {
 						Name:         aws.String(coretest.RandomName()),
 						ImageId:      aws.String("ami-123"),
 						Architecture: aws.String("x86_64"),
-						Tags:         []*ec2.Tag{{Key: aws.String(v1.LabelInstanceTypeStable), Value: aws.String("m5.large")}},
+						Tags: []*ec2.Tag{
+							{
+								Key:   aws.String(v1.LabelInstanceTypeStable),
```

Comment:
I thought we were removing these type of requirements?

Reply:
Removed them and opted for a more permissive selector